libparsec/crates/types/tests/unit/addr.rs

```diff
@@ -275,7 +298,10 @@ fn addr_with_bad_port(testbed: &dyn Testbed, #[values("NaN", "999999")] bad_port
 }
 
 #[apply(all_addr)]
-fn addr_with_bad_scheme(testbed: &dyn Testbed, #[values("xxx", "http", "https")] bad_scheme: &str) {
+fn addr_with_bad_scheme(
+    testbed: &dyn Testbed,
+    #[values("xxx", "http", "https", "parsec4", "parsec", "parsec3x")] bad_scheme: &str,
```

Comment:
```suggestion
    #[values("xxx", "http", "https", "parsec4", "parsec", "parsec3x", "par\snec3")] bad_scheme: &str,
```

So the example `par\nsec3` is tested

Reply:
This doesn't work since `par\nsec3` will be stripped of `\n` when the url is parsed, so the actual scheme becomes `parsec3`

Reply:
What I want is tested by `good_addr_with_noise_trimmed_by_url_parser`, So all good